### ISSUE TITLE
Bug: EARTHLY_SECRETS env var not working with earth command

### ISSUE THREAD

**Issue by zenyui** [earthly/earthly#229]
I believe there's a bug with the `EARTHLY_SECRETS` env var. Per docs, I should be able to define CSV secrets instead of individually passing in with the `--secret` argument.

Given a target:
```
some-target:
    RUN --secret SOME_SECRET=+secrets/SOME_SECRET echo $SOME_SECRET
```

these commands succeed
```sh
# passing in as a key/value
earth --secret SOME_SECRET=a --no-cache +code

# setting the environment inline
SOME_SECRET=a earth --no-cache --secret SOME_SECRET +some-target

# setting the env var in a separate command
export SOME_SECRET=a
earth --no-cache --secret SOME_SECRET +some-target
```
and these commands fail...
```sh
# using key/value earthly secrets inline
EARTHLY_SECRETS="SOME_SECRET=a" \
earth --no-cache +some-target

# setting env var separately as key/value
export EARTHLY_SECRETS="SOME_SECRET=a"
earth --no-cache +some-target

# setting env vars separately
export SOME_SECRET=a
export EARTHLY_SECRETS="SOME_SECRET"
earth --no-cache +some-target
```
... with the error message:
>+some-target | --> RUN echo $SOME_SECRET
+some-target | ERROR: (RUN echo $SOME_SECRET) secret SOME_SECRET not found: not found
Repeating the output of the command that caused the failure
=========================== FAILURE ===========================
+some-target *failed* | --> RUN echo $SOME_SECRET
+some-target *failed* | [no output]
+some-target *failed* | ERROR: (RUN echo $SOME_SECRET) secret SOME_SECRET not found: not found
Error: solve side effects: solve: failed to solve: rpc error: code = Unknown desc = secret SOME_SECRET not found: not found

**Comment by alexcb** [comment]
Thanks @zenyui , I believe I found a bug in the underlying cli handling library we use: https://github.com/urfave/cli/pull/1178

If we get that PR merged we'll be able to fix this, otherwise I now know how to work around it :smile: 

Thanks for the clear repro-cases.

**Comment by zenyui** [comment]
> Thanks @zenyui , I believe I found a bug in the underlying cli handling library we use: [urfave/cli#1178](https://github.com/urfave/cli/pull/1178)
> 
> If we get that PR merged we'll be able to fix this, otherwise I now know how to work around it 😄
> 
> Thanks for the clear repro-cases.

You're welcome. I'm not blocked, for now I'm just passing in the secrets explicitly. If you have a local workaround I'm happy to try it out!

**Comment by alexcb** [comment]
@zenyui we just released v0.3.2 which fixes this issue.
